notebooks/MAST/downloading_transiting_planet_data/downloading_transiting_planet_data.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<p>Inspect what column names were returned with the .columns attribute.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>